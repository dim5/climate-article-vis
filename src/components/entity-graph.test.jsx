import { render } from '@testing-library/react';

import EntityGraph from './entity-graph';

jest.mock('react-force-graph-3d', () => () => <div />);
jest.mock('react-force-graph-2d', () => () => <div />);

it('should render', () => {
  const { container } = render(<EntityGraph />);

  expect(container.innerHTML).toMatchSnapshot();
});
