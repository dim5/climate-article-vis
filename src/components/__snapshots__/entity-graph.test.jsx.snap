// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `"<div class=\\"entity-graph__GraphVisContainer-w4gmat-0 doHWDm\\"><div class=\\"entity-graph__GraphContainer-w4gmat-1 krQgec\\"><div></div></div><div class=\\"ant-radio-group ant-radio-group-solid entity-graph___StyledRadioGroup-w4gmat-2 ckXLIf\\"><label class=\\"ant-radio-button-wrapper ant-radio-button-wrapper-checked\\"><span class=\\"ant-radio-button ant-radio-button-checked\\"><input type=\\"radio\\" class=\\"ant-radio-button-input\\" value=\\"true\\" checked=\\"\\"><span class=\\"ant-radio-button-inner\\"></span></span><span><span role=\\"img\\" aria-label=\\"node-index\\" class=\\"anticon anticon-node-index entity-graph___StyledNodeIndexOutlined-w4gmat-3 bVfKNC\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"node-index\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><defs><style></style></defs><path d=\\"M843.5 737.4c-12.4-75.2-79.2-129.1-155.3-125.4S550.9 676 546 752c-153.5-4.8-208-40.7-199.1-113.7 3.3-27.3 19.8-41.9 50.1-49 18.4-4.3 38.8-4.9 57.3-3.2 1.7.2 3.5.3 5.2.5 11.3 2.7 22.8 5 34.3 6.8 34.1 5.6 68.8 8.4 101.8 6.6 92.8-5 156-45.9 159.2-132.7 3.1-84.1-54.7-143.7-147.9-183.6-29.9-12.8-61.6-22.7-93.3-30.2-14.3-3.4-26.3-5.7-35.2-7.2-7.9-75.9-71.5-133.8-147.8-134.4-76.3-.6-140.9 56.1-150.1 131.9s40 146.3 114.2 163.9c74.2 17.6 149.9-23.3 175.7-95.1 9.4 1.7 18.7 3.6 28 5.8 28.2 6.6 56.4 15.4 82.4 26.6 70.7 30.2 109.3 70.1 107.5 119.9-1.6 44.6-33.6 65.2-96.2 68.6-27.5 1.5-57.6-.9-87.3-5.8-8.3-1.4-15.9-2.8-22.6-4.3-3.9-.8-6.6-1.5-7.8-1.8l-3.1-.6c-2.2-.3-5.9-.8-10.7-1.3-25-2.3-52.1-1.5-78.5 4.6-55.2 12.9-93.9 47.2-101.1 105.8-15.7 126.2 78.6 184.7 276 188.9 29.1 70.4 106.4 107.9 179.6 87 73.3-20.9 119.3-93.4 106.9-168.6zM329.1 345.2a83.3 83.3 0 11.01-166.61 83.3 83.3 0 01-.01 166.61zM695.6 845a83.3 83.3 0 11.01-166.61A83.3 83.3 0 01695.6 845z\\"></path></svg></span>2D view</span></label><label class=\\"ant-radio-button-wrapper\\"><span class=\\"ant-radio-button\\"><input type=\\"radio\\" class=\\"ant-radio-button-input\\" value=\\"false\\"><span class=\\"ant-radio-button-inner\\"></span></span><span><span role=\\"img\\" aria-label=\\"code-sandbox\\" class=\\"anticon anticon-code-sandbox entity-graph___StyledCodeSandboxOutlined-w4gmat-4 eQQXxv\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" data-icon=\\"code-sandbox\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M709.6 210l.4-.2h.2L512 96 313.9 209.8h-.2l.7.3L151.5 304v416L512 928l360.5-208V304l-162.9-94zM482.7 843.6L339.6 761V621.4L210 547.8V372.9l272.7 157.3v313.4zM238.2 321.5l134.7-77.8 138.9 79.7 139.1-79.9 135.2 78-273.9 158-274-158zM814 548.3l-128.8 73.1v139.1l-143.9 83V530.4L814 373.1v175.2z\\"></path></svg></span>3D view</span></label></div></div>"`;
